psi4/driver/schema_wrapper.py

```diff
@@ -341,8 +351,8 @@ def re1d(mat):
         "scf_fock_b": re2d(wfn.Fa_subset("AO")),
         "scf_eigenvalues_a": re1d(wfn.epsilon_a_subset("AO", "ALL")),
         "scf_eigenvalues_b": re1d(wfn.epsilon_b_subset("AO", "ALL")),
-        # "scf_occupations_a": np.hstack(wfn.occupation_a().nph),
-        # "scf_occupations_b": np.hstack(wfn.occupation_b().nph),
+        "scf_occupations_a": re1d(sort_occs((wfn.doccpi() + wfn.soccpi()).to_tuple(), wfn.epsilon_a().nph)),
+        "scf_occupations_b": re1d(sort_occs(wfn.doccpi().to_tuple(), wfn.epsilon_b().nph)),
```

Comment:
background in case useful to others:

occupation_a/b was the only one of these that was SCF-only. and the current doccpi/soccpi will also be SCF-only. there is an occupations subset function https://github.com/psi4/psi4/blob/master/psi4/src/psi4/libmints/wavefunction.cc#L847 but it looks not so developed or parallel compared to all the other nice _subset fns. this may need to be developed to access NO occupations as qcschema.wfn expands.